### DISCUSSION TITLE
Diablo 1 palette and pcx support

### DISCUSSION THREAD

**Discussion by ghost** [diasurgical/DevilutionX#5844] [Q&A]
Hello, i'm far enough with making a [new hp bar](https://github.com/diasurgical/devilutionX/discussions/5840) that i'd like to make assets for in-game testing.

First time for me, be gentle ;)

1. Do you guys have the diablo 1 colour palette for download? preferably in a .gpl extension as i can directly plug that one into inkscape.
2. I can see that the other assets where saved as .pcx files. That file type is old so i need to figure out how to either convert my files to it or find a program for saving my work as pcx. Gimp can open pcx but i can't save as it. What do you guys do? Any recommendations?

**Comment by qndel** [comment]
I really suggest coming to discord ;)

**Reply by ghost** [reply]
Sure, i didn't know you guys had a discord :)

**Reply by AJenbo** [reply]
Link is in the support menu in game, and the included readme, and the top of the readme on github

**Comment by AJenbo** [comment]
1: They are in your diabdat.mpq, they are .pal, GIMP can load them, and since .gpl appears to stand for "GIMP palette" it can probably save to .gpl.

2. I use GIMP, it can't save .png either but it can export to both
